app/controllers/custom_issues_controller.rb

```diff
@@ -0,0 +1,15 @@
+class CustomIssuesController < ApplicationController
+  skip_before_action :verify_authenticity_token, only: :update_collapsed_ids

```

Comment:
1/ Why skip `verify_authentication_token`? Did something crashed when sending data? We need to correctly work in any scenario.
2/ I think `update_collaped_ids` should be be in the `issue_view_columns_controller.rb`, since it is related.

Reply:
1/ If I don't skip verify_authentication_token, I can't validate the AJAX request, even with the CSRF token
2/ I will move it